mkdocs.yml

```diff
@@ -48,6 +48,7 @@ nav:
       - Pipeline: docs/manifest/pipeline.en.md
     - Developing:
       - Additional AWS Resources: docs/developing/additional-aws-resources.en.md
+      - Package Addons: docs/developing/package-addons.en.md
```

Comment:
Should this be a subsection in the page of `Additional AWS Resources` instead? I don't know if the term "Addons" will immediately make sense to folks.